src/FakeItEasy/Creation/DefaultFakeCreatorFacade.cs

```diff
@@ -53,25 +53,33 @@ public object CreateFake(Type typeOfFake, Action<IFakeOptions> optionsBuilder)
         /// </returns>
         public IList<T> CollectionOfFake<T>(int numberOfFakes)
         {
+            return this.CollectionOfFake<T>(numberOfFakes, x => { });
+        }
+
+        public IList<T> CollectionOfFake<T>(int numberOfFakes, Action<IFakeOptions<T>> optionsBuilder)
+        {
             var result = new List<T>();
 
             for (var i = 0; i < numberOfFakes; i++)
             {
-                result.Add(this.CreateFake<T>(x => { }));
+                result.Add(this.CreateFake(optionsBuilder));
             }
 
             return result;
         }
 
-        public IList<object> CollectionOfFake(Type typeOfFake, int numberOfFakes)
+        public IList<object> CollectionOfFake(Type typeOfFakes, int numberOfFakes)

```

Comment:
> The parameter in A is named typeOfFake so I think we should be consistent.

Ah, yes... I changed it so that it would be the same as on the interface. I'll make it singular everywhere.